streams/src/main/java/org/apache/kafka/streams/processor/internals/InternalTopicManager.java

```diff
@@ -98,9 +98,10 @@ public InternalTopicManager(final Admin adminClient, final StreamsConfig streams
         int remainingRetries = retries;
         Set<String> topicsNotReady = new HashSet<>(topics.keySet());
         final Set<String> newlyCreatedTopics = new HashSet<>();
+        final HashSet<String> leaderNotAvailableTopics = new HashSet<>();
```

Comment:
Can we give this a more descriptive name? It might be obvious to you, but I think someone just looking at this code for the first time would not get that this actually means topics that may or may not already exist.
That said, I'm struggling to think of a good alternative...maybe `possiblyCreatedTopics` or `unknownTopics`...any better ideas?

Reply:
Good suggestion, I changed to `tempUnknownTopics`, because they are temporarily unknown topics for now. How do you think?